packages/web-react/src/components/Accordion/__tests__/Accordion.test.tsx

```diff
@@ -1,9 +1,7 @@
 import '@testing-library/jest-dom';
 import { render } from '@testing-library/react';
 import React from 'react';
-import { classNamePrefixProviderTest } from '../../../../tests/providerTests/classNamePrefixProviderTest';
-import { restPropsTest } from '../../../../tests/providerTests/restPropsTest';
-import { stylePropsTest } from '../../../../tests/providerTests/stylePropsTest';
+import { classNamePrefixProviderTest, restPropsTest, stylePropsTest } from '@local/tests';
```

Comment:
**todo (if-minor):** We need to setup the correct import order. Maybe @curdaj can help with it.